src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -378,3 +431,40 @@ public async Task StartAsync(CancellationToken cancellationToken = default)
         public Task StopAsync(CancellationToken cancellationToken = default) => innerHost.StopAsync(cancellationToken);
     }
 }
+
+internal sealed class LaunchSettings
+{
+    [JsonPropertyName("profiles")]
+    public Dictionary<string, LaunchProfile> Profiles { get; set; } = new Dictionary<string, LaunchProfile>();
+}
+
+internal sealed class LaunchProfile
```

Comment:
We have identical code in `Aspire.Hosting` ... shared code?

Reply:
Good idea. Done